content/product-engineering/engineering/cloud/security/index.md

```diff
@@ -196,7 +196,7 @@ In an effort to work closely with our teammates and shift security focus more in
   <tr>
    <td>Cloud
    </td>
-   <td>Extensibility
+   <td>Growth and Integrations
```

Comment:
Should we remove `Growth` in the above line 189?

Reply:
I think this has an impact on the Security team and that they should do this. I'll communicate as such.